src/modules/DongleDriver.ts

```diff
@@ -62,8 +66,10 @@ export class DongleDriver extends EventEmitter {
       this._device = device
       console.debug('opening')
 
-      await this._device.open()
-      await this._device.reset()
+      console.debug('initializing')
+      if (!device.opened) {
+        throw new Error('Illegal state - device not opened')
```

Comment:
@steelbrain  The driver has a custom error type for state exceptions - DriverStateError - could you use that instead please?